docs/BR.md

```diff
@@ -1468,7 +1468,9 @@ The CA SHALL verify that the Delegated Third Party's personnel involved in the i
 
 ### 5.4.1 Types of events recorded
 
-The CA and each Delegated Third Party SHALL record details of the actions taken to process a certificate request and to issue a Certificate, including all information generated and documentation received in connection with the certificate request; the time and date; and the personnel involved. The CA SHALL make these records available to its Qualified Auditor as proof of the CA’s compliance with these Requirements.
+The CA and each Delegated Third Party SHALL record events related to the security of Certificate Systems, Certificate Management Systems, Root CA Systems and Delegated Third Party Systems, especially events and details of the actions taken to process a certificate request and to issue a Certificate.
```

Comment:
Should we include a reference or pointer to the definitions of these in the NCSSRs?

Reply:
> 
> 
> Should we include a reference or pointer to the definitions of these in the NCSSRs?

Yes, this makes sense.

Reply:
Should we include a "catch all" phrase in section 1.6.1 pointing to definitions in the NCSSRs and possibly the EV Guidelines, or should we add specific references in section 5?

Reply:
I had been leaning towards adding them in section 5, but after giving it further thought I think 1.6.1 makes more sense as that seems an absence that is worth addressing regardless and may make it easier to both use and improve these NCSSRs-defined terms within the BRs in the future.